web-app/.eslintrc.js

```diff
@@ -73,6 +73,7 @@ module.exports = {
 
   // add your custom rules here
   rules: {
+    'no-unsafe-assignment': 'off',
```

Comment:
It is risky to disable this rule. Where did you have a problem?

Reply:
Can be deleted. Didn't use anywhere finally.